Core/include/Acts/EventData/ParameterSet.hpp

```diff
@@ -490,21 +490,22 @@ class ParameterSet {
    * vector of parameter values of this ParameterSet object and \f$\vec{v}^0\f$
    * is the full parameter value vector.
    *
-   * @param boundParameters Vector of bound parameters
-   * @note Constraint and cyclic parameter value ranges of @p boundParameters
-   * are not tested.
-   * @note It is not tested whether @p boundParameters is at the same reference
-   * object
+   * @param BoundTrackParameters Vector of bound parameters
+   * @note Constraint and cyclic parameter value ranges of @p
+   * BoundTrackParameters are not tested.
+   * @note It is not tested whether @p BoundTrackParameters is at the same
+   * reference object
    *
    * @return vector containing the residual parameter values of this
    * ParameterSet object
    *         with respect to the given full parameter vector
    *
    * @sa ParameterSet::projector
    */
-  ParameterVector residual(const ActsVectorD<kSizeMax>& boundParameters) const {
+  ParameterVector residual(
+      const ActsVectorD<kSizeMax>& BoundTrackParameters) const {
```

Comment:
don't capitalize the first letter of function parameters :)

Reply:
also you seem to have applied a different clang-format than we use, this won't pass.